docs/config.json

```
@@ -476,7 +476,19 @@
         {
           "title": "Access Graph",
           "slug": "/access-graph/",
-          "forScopes": ["team"]
+          "forScopes": [
+            "team",
+            "enterprise"
+          ],
+          "entries": [
+            {
+              "title": "Getting Started On-Premises",
```

```suggestion
              "title": "Self-Hosting TAG",
```
Since the instructions can also apply to cloud hosted compute infrastructure, unless I'm missing something?

---

That's true, this tries to communicate using TAG with a self-hosted cluster (vs Teleport Cloud).

I liked "Getting Started" since we use that for a bunch of other features and it is an obvious first place to click.

"Self-Hosting TAG" perhaps fails to communicate upfront that this is required when running against a self-hosted Teleport cluster. I wonder if there's a way to phrase the entry so it roughly matches the full page title:

> Run Teleport Access Graph on Self-Hosted Clusters

while being a bit shorter (which we probably want to avoid clunky menu items)